tools/ci/docbuild_install.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```

Comment:
Coach is complaining about these she bangs being misssing

Reply:
These are only for use with source and no standalone scripts, so no shebang.